Edge/cie 4179 update install takes more time (#67)
SoftwareAG/c8y-releasenotes: content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md

Starting with Cumulocity IoT Edge Release 10.15, accessing the Edge appliance remotely from the Cumulocity IoT tenant using the Telnet protocol has been removed. Software AG recommends you to use the SSH protocol instead.

> ### Delay in installation and update

Reviewer: - [ ] Is the heading right by calling it as a "Delay"? We may have to find better wording.

Author: Updated the heading to "Installation and update processes take more time"

> ### Installation and update processes take more time

Reviewer 2: ```suggestion
### Installation and update processes take longer
```

> Installation and Update processes perform security hardening of the operating system and other components, due to which these processes take longer (approximately 50% higher) than the earlier versions.

Reviewer 2: ```suggestion
The installation and update processes perform security hardening of the operating system and other components.
As a result, these processes take approximately 50% longer than in earlier versions.
```
